packages/@okta/vuepress-site/docs/api/resources/system_log/index.md

```diff
@@ -396,6 +396,7 @@ Among other operations, this response object can be used to scan for suspicious
 | Property               | Description                                                                                                                                                                                                                         | DataType                        | Nullable | MinLength | MaxLength |
 | ----------             | --------------------------------------------------------------                                                                                                                                                                      | ---------------                 | -------- | --------- | --------- |
 | authenticationProvider | The system that proves the identity of an actor using the credentials provided to it                                                                                                                                                | String                          | TRUE     |           |           |
+| authenticationStep     | The zero-based step number in the authentication pipeline (`0` - primary factor authn, `1` - secondary factor authn, etc).                                                                                                          | Integer                         | TRUE     |           |           |
```

Comment:
Looks like this is only being used in an event currently fired by [DeviceTrust](https://github.com/okta/okta-core/blob/master/components/platform/device-trust/core/src/main/java/com/okta/monolith/platform/devicetrust/events/DeviceTrustEventHelperImpl.java#L339)

Also, looks like Passwordless might use it in the future which is owned by our team MFAPlatform

Description looks good to me
cc'ing other stakeholders

@mohammadrahimi-okta @sshen-okta @srinivasanagandla-okta

Reply:
I wasn't aware of `authenticationStep`. As Ajay it seems to be used by device trust and I see all the usages have the value as `0`. 

I think we should remove reference to it for now and doc it when we are ready at a product level.

Rest of it looks good to me.

@tthompson-okta FYI...

Reply:
Well, it shows up in the API response and the existing examples show it so for completeness I tend to think we should document it, but say that it is currently unused and always `0`. When the behavior changes we should then correct the docs.

As an aside, which is the correct team to handle the behaviour change in the platform?

Reply:
@bobtiernay-okta I guess it is @okta/idx team. @tthompson-okta does that sound right to you?

I am still not sure if we want to document `authenticationStep` given that it could change.

Reply:
🚢